app/src/main/res/layout/profile_edit_activity.xml

```diff
@@ -12,228 +12,235 @@
       type="org.oppia.android.app.settings.profile.ProfileEditViewModel" />
   </data>
 
-  <androidx.constraintlayout.widget.ConstraintLayout
+  <ScrollView
```

Comment:
Do the `Reformat Code`. Check our wiki guide here - https://github.com/oppia/oppia-android/wiki/Ktlint-Guide#macros
We will be updating the wiki specifying for XML code too., as of now it says for kotlin only, but do the same for XML.

Reply:
DONE